tests/Application/Actions/Donations/CreateTest.php

```diff
@@ -231,7 +231,10 @@ public function testSuccessWithStripeAccountIDMissingInitiallyButFoundOnRefetch(
         $donation->setCharityFee('0.38'); // Calculator is tested elsewhere.
         $donation->getCampaign()->getCharity()->setStripeAccountId(null);
 
-        $fundingWithdrawalForMatch = new FundingWithdrawal();
+        $campaignFunding = new CampaignFunding();
+        $campaignFunding->setCurrencyCode('GBP');
+        $campaignFunding->setAmountAvailable('8.00');
+        $fundingWithdrawalForMatch = new FundingWithdrawal($campaignFunding);
         $fundingWithdrawalForMatch->setAmount('8.00'); // Partial match
         $fundingWithdrawalForMatch->setDonation($donation);
```

Comment:
Looks like this exact sequence of three lines is repeated 5 times. Can we extract to a function so we don't have to edit 5 copies next time?

```
$fundingWithdrawalForMatch = new FundingWithdrawal($campaignFunding);
$fundingWithdrawalForMatch->setAmount('8.00'); // Partial match
$fundingWithdrawalForMatch->setDonation($donation);
```